File: showtime-service/src/main/java/com/example/showtime_service/model/Showtime.java
package com.example.showtime_service.model;

import com.google.cloud.Timestamp;
import lombok.Data;

@Data
public class Showtime {
    private String showtimeId;
    private String movieId;
    private String theaterId;
    private String roomId;
    private Timestamp startTime; // ✅ Đổi từ String sang Timestamp
    private double ticketPrice;
    private int totalTickets;
    private int availableTickets;
}

File: AndroidApp/app/src/main/java/com/btlcnpm/androidapp/data/model/Showtime.kt
package com.btlcnpm.androidapp.data.model

// Dựa trên ShowtimeResponse.java từ showtime-service
data class Showtime(
    val showtimeId: String,
    val movieId: String,
    val theaterId: String,
    val roomId: String,
    val startTime: String, // Chuỗi ISO 8601 (ví dụ: "2025-10-26T19:00:00Z")
    val ticketPrice: Double,
    val availableTickets: Int
)